Sources/CardPayments/APIRequests/CheckoutOrdersAPI.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+#if canImport(CorePayments)
+import CorePayments
+#endif
+
+/// This class coordinates networking logic for communicating with the v2/checkout/orders API.
+///
+/// Details on this PayPal API can be found in PPaaS under Merchant > Checkout > Orders > v2.
+class CheckoutOrdersAPI {
+    
+    // MARK: - Private Propertires
+    
+    private let coreConfig: CoreConfig
+    
+    // MARK: - Initializer
+    
+    init(coreConfig: CoreConfig) {
+        self.coreConfig = coreConfig
+    }
+    
+    // MARK: - Internal Methods
+        
+    func confirmPaymentSource(clientID: String, cardRequest: CardRequest) async throws -> ConfirmPaymentSourceResponse {
+        let apiClient = APIClient(coreConfig: coreConfig)
+        
+        let confirmData = ConfirmPaymentSourceRequest(cardRequest: cardRequest)
+        
+        let base64EncodedCredentials = Data(clientID.appending(":").utf8).base64EncodedString()
+        
+        // TODO: - Move JSON encoding into custom class, similar to HTTPResponseParser
+        let encoder = JSONEncoder()
+        encoder.keyEncodingStrategy = .convertToSnakeCase
+        let body = try encoder.encode(confirmData)
+        
+        let restRequest = RESTRequest(
+            path: "/v2/checkout/orders/\(cardRequest.orderID)/confirm-payment-source",
+            method: .post,
+            headers: [
+                .contentType: "application/json", .acceptLanguage: "en_US",
+                .authorization: "Basic \(base64EncodedCredentials)"
+            ],
+            queryParameters: nil,
+            body: body
+        )
+        
+        let httpResponse = try await apiClient.fetch(request: restRequest)
+        return try HTTPResponseParser().parse(httpResponse, as: ConfirmPaymentSourceResponse.self)
+    }
```

Comment:
So this class is strictly for confirm-payment-source endpoint, right?

Reply:
Based on the class description I think this would be anything under `v2/checkout/orders` in PPaaS. Right now that's just `confirm-payment-source` in our SDK but it seems like that's the intent of how we expect this class to grow. Is my  understanding correct @scannillo?

Reply:
Exactly what Jax said! Huge kudos to Steven for coming up with the idea of separate classes per API 🎊